main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -32,51 +34,65 @@ import scala.collection.mutable
   */
 object Resolver extends Phase[NamedAst.Root, ResolvedAst.Root] {
 
+  private type SigLookup = Map[Name.NName, Map[String, NamedAst.Sig]]
```

Comment:
Since we dont really use type aliases for other things, I think this could just be inline.